include/kunit

```diff
@@ -172,6 +172,9 @@ need_kconfig:
 - USB4_DMA_TEST: m
 - UNICODE_NORMALIZATION_SELFTEST: m
 - UNICODE: m
+- TSNEP_SELFTESTS: y
+- TSNEP: m
+- PTP_1588_CLOCK_OPTIONAL: m
```

Comment:
this need be put after "# below are kunit test configs of bool type" line

also refer to IOMMUFD_TEST, put dependency kconfigs before it, and have empty line before/after.
```
- DEBUG_KERNEL: y
- FAULT_INJECTION: y
- IOMMUFD
- IOMMUFD_TEST: y
```

Reply:
pls also move its dependency together. Since it has 3 kconfigs together, pls add empty line before/after them.